controllers/package_controller.go

```diff
@@ -80,12 +82,14 @@ func RegisterPackageReconciler(mgr ctrl.Manager) (err error) {
 	}
 	puller := artifacts.NewRegistryPuller()
 	bundleManager := bundle.NewBundleManager(log, discovery, puller)
```

Comment:
I wonder if the BundleManager should be passed a BundleClient instead of a k8s client for its `IsActive` and `ActiveBundle` methods. Or alternatively at instantiation (though I think that might break the abstraction we were going for.)

In any event, the larger point here, that I'd like to get your feedback on is this: We now have two different constructs that are able to retrieve the active bundle. I'd prefer there to be one place to get that information. Meaning that the BundleManager should probably have its IsActive and ActiveBundle methods removed... or something?

Reply:
Agreed, but as a follow-up. I needed to make this PR as small as possible.